### ISSUE TITLE
Mandatory alt attribute missing from IMG tags

### ISSUE THREAD

**Issue by berndca** [elm-lang/elm-lang.org#474]
I ran the converted html through a html validator. The validator reported errors for the images: missing  alt attributes. IMO this is primarily an accessibility issue for users depending on screen readers and therefore should be corrected.
